e2e/test/access-key.ts

```diff
@@ -0,0 +1,31 @@
+import * as assert from "assert";
```

Comment:
Remove this file. Only add the meaningful tests and code. Do not populate the PR with unnecessary changes.